components/addForm.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+
+export default function AddCharacterForm() {
+  return (
+    <StyledContainer>
+      <StyledInputField
+        type="text"
+        maxLength={20}
+        placeholder="Name"
+      ></StyledInputField>
+      <StyledTextarea
+        type="text"
+        maxLength={255}
+        placeholder="Enter here your information...
+        Attributes, Skills, Characteristics, Items, Character-Story, etc."
+      ></StyledTextarea>
```

Comment:
```suggestion
      <StyledInputField
        required
        type="text"
        maxLength={20}
        name="name"
        placeholder="Name"
      />
      <StyledTextarea
        required
        type="text"
        maxLength={255}
        name="information"
        placeholder="Enter here your information...
        Attributes, Skills, Characteristics, Items, Character-Story, etc."
      />
```

Reply:
Add required so users can't leave the textfield empty. Also add a name tag for later usage in functionality.